<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DesignerRootWidget</class>
 <widget class="QWidget" name="DesignerRootWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="layoutWidgetBar"/>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <layout class="QVBoxLayout" name="layoutForm">
        <item>
         <widget class="QMdiArea" name="mdiArea"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_3">
       <layout class="QVBoxLayout" name="layoutBottom">
        <item>
         <widget class="QTabWidget" name="tabs"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_4">
       <layout class="QVBoxLayout" name="layoutObjectInspector"/>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_5">
       <layout class="QVBoxLayout" name="layoutProperty"/>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
